CHANGELOG.md

```diff
@@ -0,0 +1,101 @@
+# Changelog
+
+[npm history][1]
+
+[1]: https://www.npmjs.com/package/nodejs-storage?activeTab=versions
+
+## v2.0.0
+
+**This release has breaking changes**.  This release has a few notable breaking changes.  Please take care when upgrading!
+
+### require syntax changes
+The import style of this library has been changed to support [es module](https://nodejs.org/api/esm.html) syntax.  This provides both forward compatibility with es modules, and better supports the TypeScript and Babel ecosystems.  As a result, the import syntax has changed:
+
+#### Old Code
+```js
+const storage = require('@google-cloud/storage')();
+// or...
+const Storage = require('@google-cloud/storage');
+const storage = new Storage({
+  // config...
+});
+```
+
+#### New Code
+```js
+const {Storage} = require('@google-cloud/storage');
+const storage = new Storage({
+  // config...
+});
+```
+
+### `bucket.upload` method removed
+To better support a variety of HTTP clients, the `bucket.upload` method has been removed. It can be replaced with your favorite HTTP client.
+
+#### Old Code
+```js
+bucket.upload('https://example.com/images/image.png', function(err, file, res) {
+  // handle upload...
+});
+```
+
+#### New Code
+
+```js
+const request = require('request');
+let readStream = request(url).createReadStream();
```

Comment:
There isn't a `createReadStream()` method. This should just be:

```js
request(url).pipe(file.createWriteStream());
```

Reply:
Thank you! Can you check what I have there now?